### ISSUE TITLE
Filter Games list by Console in the Hakchi window (GUI)

### ISSUE THREAD

**Issue by FoyzM** [ClusterM/hakchi2#1007]
**Note**: I'm not sure how to re-open closed posts, so I'm requesting again. I'm talking about the games management console, **NOT** the kernel output (folders feature)

Would help keep it clean for those that don't want to mix games on the different consoles. :)
I know you can now have a separate list for SNES and NES uploads, but you still have to manually manage those lists. So I'll ask again, is it possible to add filter options so the games management console can hide by console type, like master system, genesis, neo-geo etc. Personally, I plan on purchasing every iteration of the mini-consoles and would like to add all games for each console type...

**Comment by JohnSid45** [comment]
I just sort them manually, but it's not too hard to do.  Most of them have different thumbnail icons by console, so when I add new games I drop them all into folders by console type, then let hakchi sort the console folders automatically.  It'd be nice to have this sort of filter in the interface where you pick the games, though.

**Comment by Drakonas** [comment]
To clarify @FoyzM Do you mean the list in the actual Hakchi2 program, and not the outputted organization on the console?

**Comment by gingerbeardman** [comment]
I'm pretty sure he means the "Select Games" panel. 

It's an interface meant for managing tens of games on one system, and doesn't work so well when you're dealing with hundreds or thousands of games from dozens of systems. 

![140882ea-3fa7-4134-b105-5909ab64538f](https://user-images.githubusercontent.com/49612/32582198-7df6e21a-c4e5-11e7-80ac-16aed761ae0f.jpeg)

**Comment by Drakonas** [comment]
yeah @gingerbeardman I just wanted clarification from @FoyzM . His original ticket gave the vibe of talking about the organization on the console itself, and now I see he may have been talking about the GUI of the Hakchi window itself, which I agree: this feature would be nice by those standards.

**Comment by FoyzM** [comment]
@drakonas yes that's what I meant. @gingerbreadman I see, perhaps a way to create separate select games lists, something like a load by console? So Hakchi2 only loads the games for a particular console? Then it would only be managing a couple of hundred at a time. Hakchi2 already does this when selecting between SNES and NES options. I've also been using Cylums ROM Set which is around 2000 games, obviously you can never upload that many, but trying to manage the select games is quite time consuming. Having said that Hakchi2 actually handles the sets extremely well.

**Comment by cryptolute** [comment]
From someone who is trying to do a ultimate USB-Host build, I would like to 2nd, 3rd and 999,999 this request. I find myself wishing that I could sort the game list by console type and then have the ability to select/deselect based on console type.

We really need this feature combined with region. The program already does a fantastic job at identifying games by known crc, it would be reasonably trivial to add a sort feature that would organize based on console type/region since these are identified already by hakchi.

Thank you clusterm!
